kalite/distributed/static/js/distributed/rating/models.js

```diff
@@ -0,0 +1,18 @@
+var Backbone = require("../base/backbone");
+
+var RatingModel = Backbone.Model.extend({
+    urlRoot: "/api/content_rating"
+});
+
+var ContentRatingCollection = Backbone.Collection.extend({
+    parse: function(response) {

```

Comment:
Is this necessary?

c.f. in Backbone-tastypie:
```
	Backbone.Collection.prototype.parse = function( data ) {
		if ( data && data.meta ) {
			this.meta = data.meta;
		}

		return data && data.objects || data;
	};
```

Should already be happening.